GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -585,6 +585,9 @@ private void PostFetchJob(List<string> packIndexes)
 
                     using (ITracer activity = this.context.Tracer.StartActivity("TryWriteMultiPackIndex", EventLevel.Informational, Keywords.Telemetry, metadata: null))
                     {
+                        string midxLockFile = Path.Combine(this.context.Enlistment.GitPackRoot, "multi-pack-index.lock");

```

Comment:
Should we have these lock files as named constants, like the post fetch lock file?